api/src/main/kotlin/edu/wgu/osmt/config/AppConfig.kt

```diff
@@ -31,6 +31,12 @@ class AppConfig(
     @Value("\${app.loginSuccessRedirectUrl}")
     val loginSuccessRedirectUrl: String,
 
+    @Value("\${app.userName}")
+    val userName: String,
+
+    @Value("\${app.userIdentifier}")
```

Comment:
I wonder if we should provide a default for these

@Value("\${app.userIdentifier:email}")